pysrc/bytewax/connectors/kafka.py

```diff
@@ -0,0 +1,106 @@
+"""Connectors for [Kafka](https://kafka.apache.org).
+
+"""
+from typing import Dict, Iterable
+
+from confluent_kafka import Consumer, KafkaError, OFFSET_BEGINNING, TopicPartition
+from confluent_kafka.admin import AdminClient
+
+from bytewax.inputs import CustomPartInput
+
+
+def _stateful_read_part(consumer):
+    while True:
+        msg = consumer.poll(0.001)  # seconds
```

Comment:
This seems to greatly alleviate the spinning problem on the kafka input, but I think it's can still be present in some other kind of inputs, it would be good to do this at the timely level probably. But that's for another PR.

Reply:
Yes. Async IO will be a big project, but hopefully this will make it acceptable for the time being.